pkg/api/types.go

```diff
@@ -1368,6 +1368,13 @@ type NodeSpec struct {
 	Unschedulable bool `json:"unschedulable,omitempty"`
 }
 
+type Heartbeat struct {
+	// Time when this heartbeat was generated.
```

Comment:
I think that you also need ObjectMeta - similarly to what we have in binding:
https://github.com/gmarek/kubernetes/blob/heartbeat_api/pkg/api/types.go#L1568